analyses/molecular-subtyping-EPN/01-make_notebook_RNAandDNA.py

```diff
@@ -62,10 +56,13 @@ def group_disease(primary_site):
 # sample_id is common between both  datafarmes and also unique between RNA and DNA. 
 # Some DNA BSID's are missing for the corresponding RNA samples
 EP_rnaseq_WGS = EP_rnaseq_samples.merge(WGS_dnaseqsamples, 
-                                        on = ["sample_id", "Kids_First_Participant_ID"], 
+                                        on = ["sample_id", "Kids_First_Participant_ID","primary_site"], 
```

Comment:
I assume here that `primary_site` does not change anything about the way the tables are matched, but is in the merge statement to prevent creating a duplicate column?

Reply:
Right, I added primary_site here primarily to avoid the _x and _y duplicated columns,

```
Index(['Kids_First_Biospecimen_ID_RNA', 'Kids_First_Participant_ID',
       'sample_id', 'primary_site_x', 'Kids_First_Biospecimen_ID_DNA',
       'primary_site_y'],
      dtype='object')
```